client/src/Views/Home/Prospects/ProspectsUpload.js

```diff
@@ -0,0 +1,197 @@
+          marginTop: theme.spacing(6),
+          marginBottom: theme.spacing(6),
+          padding: theme.spacing(3),
+        },
+      },
+      upload : {
+        marginTop: theme.spacing(5),
+      },
+      uploadButton : {
+        backgroundColor : `${colors.lightGreen}`,
+        padding: theme.spacing(2),
+        
+        color : `${colors.white}`,
+        '&:hover': {
+            backgroundColor:`${colors.white}`,
+            color : `${colors.lightGreen}`,
+            border: "1px solid #4FBE75",
+        }
+      },
+      
+      dropzone : {
+          backgroundColor : `${colors.gray}`,
+          borderRadius: 7,
+          paddingTop: theme.spacing(3),
+          paddingBottom: theme.spacing(3),
+          marginBottom: theme.spacing(5),
+          border: "1px dashed #bbbaba",
+          width : "70%",
+          '&:hover': {
+            backgroundColor: "#bbbaba",
+
+        }
+      },
+      resultMessage : {
+        paddingTop: theme.spacing(3),
+      }
+  });
+
+
+  class ProspectsUpload extends Component {
+    
+    state = {
+        file : null,
+        uploadSuccess: false,
+        fileMessage : '',
+        uploadResultMessage : ''
+    }
+
+    onsubmit = async (e) => {
+        console.log(this.state.file);
+        let file = this.state.file;
+        let formData = new FormData();
+        formData.append('file', file);
+
+        try{
+            const res = await uploadProspectCsv(formData);
+            if(res.status==200) {
+                this.setState({
+                    uploadResultMessage : 
+                    "Successfully uploaded " + res.data.successCount + " records. \n\n" +
+                    "Skipped " + res.data.skippedCount + " records."
+                })
+            }
+        } catch(error) {
+            this.setState({
+                uploadResultMessage : 
+                "Error occured. File not uploaded."
+            })
+            console.log(error);
+        }
+    }
+
+    handleFileSelect =  (e) => {
+        this.setState({
+            uploadResultMessage : ""
+        })
+        let file = e[0];
+        if(file.type !== 'text/csv') {
+            console.log(file.mimetype);
+            this.setState({
+                uploadResultMessage : "File must be of type *.csv"
+            });
+        }
+        try{
+            this.setState({file : file,
+            fileMessage : '\"' + file.name + '\"' + ' selected.'});
+        }catch (error) {
+            this.setState({
+                uploadResultMessage : "Error occured. File not uploaded."
+            });
+        }
+        
+    }
+
+    //classes = useStyles();
+    render () {
+        const { classes } = this.props;
+        return (
+            <Grid className={classes.root} container spacing={0}>
+            <Grid item xs={3}>
+                <ProspectSidebar
+                //onChange={this.handleFieldChange}
+                />
+            </Grid>
+            <Grid item xs={9}>
```

Comment:
We'd probably want to add this specific route inside the Prospect page so that you don't have to repeat the sidebar up there